xml/System.IO/Path.xml

```diff
@@ -1911,7 +1911,7 @@ Paths are resolved by calling the <xref:System.IO.Path.GetFullPath%2A> method be
       <Parameters />
       <Docs>
         <summary>Returns the path of the current user's temporary folder.</summary>
-        <returns>The path to the temporary folder, ending with a backslash.</returns>
+        <returns>The path to the temporary folder, ending with a `DirectorySeparatorChar`.</returns>
```

Comment:
You can add a semantic reference to the `DirectorySeparatorChar` field:

```suggestion
        <returns>The path to the temporary folder, ending with a  <see cref="DirectorySeparatorChar"/>.</returns>
```

Reply:
Ah, that's how you do it, thank you!